### PR TITLE
Home page statistics being displayed correctly

### PR DESCRIPTION
**Pull request by zhongh**
screen shot 2016-05-16 at 12 40 07 am

Display is correct. Suggestions on modifying how the code is written?

### CODE REVIEW

**Comment by zhongh** [comment]
For reference, the queries are:
- Number of DCO Members (That's the way I could think of):

```
        SELECT (COUNT(DISTINCT ?s) AS ?count)
        WHERE
        {
            ?s a foaf:Person .
            ?s <http://vivo.mydomain.edu/ns#networkId> ?id .
        }
```
- Publications that are contributions to the DCO:

```
 SELECT (COUNT(DISTINCT ?publication) AS ?count)
WHERE {
    { ?publication a bibo:Article . }
    UNION { ?publication a bibo:Book . }
    UNION { ?publication a bibo:DocumentPart . }
    UNION { ?publication a dco:Poster . }
    UNION { ?publication a bibo:Thesis . }
    ?publication dco:isContributionToDCO ?isDcoPublication .
    FILTER (lcase(str(?isDcoPublication)) = "yes")
}
```
- Number of Datasets:

```
        SELECT (COUNT(DISTINCT ?s) AS ?count)
        WHERE
        {
        ?s a vivo:Dataset .
        ?s vitro:mostSpecificType vivo:Dataset .
        }
```
-  Number of Projects:

```
        SELECT (COUNT(DISTINCT ?s) AS ?count)
        WHERE
        {
        ?s a vivo:Project .
        ?s vitro:mostSpecificType vivo:Project .
        }
```
- Number of Field Studies:

```
        SELECT (COUNT(DISTINCT ?s) AS ?count)
        WHERE
        {
        ?s a dco:FieldStudy .
        ?s vitro:mostSpecificType dco:FieldStudy .
        }
```

**Comment by mrpatrickwest** [comment]
Can you take a screen shot of the whole front page of VIVO with the new statistics?

Also, please verify that this all works in Chrome, Safari, Firefox and IE (if you can)

Problem is, we've got two branches on dco.tw and one branch already on udco.tw so difficult to push this to test. Probably should push to dco.tw at some point, we can coordinate with Ahmed.

**Comment by zednis** [comment]
question: is this part of the query necessary?

```
{ ?publication a bibo:Article . }
    UNION { ?publication a bibo:Book . }
    UNION { ?publication a bibo:DocumentPart . }
    UNION { ?publication a dco:Poster . }
    UNION { ?publication a bibo:Thesis . }
```

Are there resources marked as contributions to the DCO that we do not want to return?

**Comment by mrpatrickwest** [comment]
It's the same constraint we have in the elasticsearch list query for publications. There are presentations, reports, that we don't want included. Not 100% sure that those are necessarily marked as contributions to the DCO, but wanted to make it consistent between the browser and the stats.

**Comment by zhongh** [comment]
Works for Safari and Chrome (unable checked for IE/Firefox):
screen shot 2016-05-16 at 1 19 38 pm
I can coordinate with Ahmed to see how we can put it on dco.tw for testing, jointly with the slideshow

**Comment by mrpatrickwest** [comment]
Is there a way we can eliminate all that space between the bottom of the image caption and the statistics. Quite a bit of dead space there.

**Comment by mrpatrickwest** [comment]
found it ... I'm pushing a change

**Comment by zhongh** [comment]
👍

File: themes/dco/templates/page-home.ftl [hunk 1]
```diff
@@ -22,7 +22,8 @@
             <#include "geoFocusMapScripts.ftl">
         </#if>
         <script type="text/javascript" src="${urls.base}/js/homePageUtils.js?version=x"></script>
-        <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js"></script>
+        <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js"></script>
```
Comment (zhongh) [inline themes/dco/templates/page-home.ftl:25]: @mrpatrickwest I tried to replace this line to something like

```
<script type="text/javascript" src="/vivo/themes/dco/js/facetview2/vendor/jquery/1.7.1/jquery-1.7.1.min.js"></script>
```

but the JS won't load on my local machine and browser gives error: "Failed to load resource: the server responded with a status of 404 (Not Found)"

Comment (zhongh) [inline themes/dco/templates/page-home.ftl:25]: I figured that we can either add jquery.min.js to productMods, or keep importing it from an external source like this.

**Comment by zhongh** [comment]
Tested on localhost and on dco.tw. Only question from me is in the in-line comment.

**Comment by mrpatrickwest** [comment]
With the new datasets in place you'll need to change your dataset query to look for dcodata:Dataset now instead of vivo:Dataset.

**Comment by mrpatrickwest** [comment]
prefix dcodata is http://info.deepcarbon.net/data/schema#

**Comment by zhongh** [comment]
@mrpatrickwest new Dataset prefix tested on local and dco.tw.

**Comment by mrpatrickwest** [comment]
I like the way you organized the queries ... very nice.

👍 

@zednis if you get a chance could you take a quick look. Then we can get this into production and work on the slideshow this afternoon

**Comment by mrpatrickwest** [comment]
@am-e can you also take a look at this pull request and give your thumbs up, make comments, whatever

**Comment by mrpatrickwest** [comment]
I think we're good to go ... just waiting on feedback from @am-e

**Pull request merged**